src/main/java/com/rapid7/client/dcerpc/mslsad/messages/LsarLookupNamesRequest.java

```diff
@@ -0,0 +1,147 @@
+ *<p><strong>ReferencedDomains: </strong>On successful return,
+ *contains the domain information for the domain to which each security principal
+ *belongs. The domain information includes a NetBIOS domain name and a domain SID
+ *for each entry in the list.</p>
+ *<p><strong>TranslatedSids: </strong>On successful return, contains
+ *the corresponding SID forms for security principal names in the <em>Names</em>
+ *parameter. It MUST be ignored on input.</p>
+ *<p><strong>LookupLevel: </strong>Specifies what scopes are to be
+ *used during translation, as specified in section <a href="https://msdn.microsoft.com/en-ca/library/cc234458.aspx">2.2.16</a>.</p>
+ *<p><strong>MappedCount: </strong>On successful return, contains
+ *the number of names that are translated completely to their SID forms. This
+ *parameter has no effect on message processing in any environment. It MUST be
+ *ignored on input.</p>
+ *<p><strong>Return Values: </strong>The following table contains a
+ *summary of the return values that an implementation MUST return, as specified
+ *by the message processing shown after the table.</p>
+ * <a href="https://msdn.microsoft.com/en-ca/library/cc234495.aspx"> Table</a>
+ *<dl>
+ *<dd>
+ *
+ *
+ *</dd></dl>
+ *<p>The behavior required when receiving an LsarLookupNames
+ *message MUST be identical to that when receiving an <a href="https://msdn.microsoft.com/en-ca/library/cc234494.aspx">LsarLookupNames2</a> message,
+ *with the following exceptions:</p>
+ *<p>
+ *Elements in the TranslatedSids output structure do not contain a <strong>Flags</strong>
+ *field.</p>
+ *<p>
+ *Due to the absence of the <em>LookupOptions</em> and <em>ClientRevision</em>
+ *parameters, the <a href="https://msdn.microsoft.com/en-ca/library/cc234422.aspx#gt_ae65dac0-cd24-4e83-a946-6d1097b71553">RPC server</a>
+ *MUST assume that <em>LookupOptions</em> is 0 and <em>ClientRevision</em> is 1.</p>
+ *<p>
+ *The server MUST return STATUS_ACCESS_DENIED if neither of the
+ *following conditions is true:</p>
+ *
+ *The RPC_C_AUTHN_NETLOGON security provider (as specified in <a href="https://msdn.microsoft.com/en-ca/library/cc243560.aspx">[MS-RPCE]</a>
+ *section <a href="https://msdn.microsoft.com/en-ca/library/cc243578.aspx">2.2.1.1.7</a>)
+ *and at least RPC_C_AUTHN_LEVEL_PKT_INTEGRITY authentication level (as specified
+ *in [MS-RPCE] section <a href="https://msdn.microsoft.com/en-ca/library/cc243867.aspx">2.2.1.1.8</a>)
+ *were used in this <a href="https://msdn.microsoft.com/en-ca/library/cc234422.aspx#gt_8a7f6700-8311-45bc-af10-82e10accd331">RPC</a>
+ *message.
+ *
+ *The PolicyHandle was granted POLICY_LOOKUP_NAMES access.
+ *
+ *                </div>
+ *            </div>
+ *        </div>
+ */
+
+public class LsarLookupNamesRequest extends RequestCall<LsarLookupNamesResponse> {
+    private final static short OP_NUM = 14;
+    private final static int LSA_LOOKUP_NAMES_ALL = 0x1;
```

Comment:
The LSAT doesn't seem to have this. I assume it's the same as the first one but called differently?
 https://msdn.microsoft.com/en-us/library/cc234458.aspx